Makefile

```diff
@@ -2,29 +2,33 @@
 # replacing reference images via env variable.
 include pkg/project/testdata/env
 
-generate-ci: clean generate-eventing-ci generate-serving-ci
-.PHONY: generate-ci
+all: eventing serving operator client
+.PHONY: all
 
-generate-ci-no-clean: generate-eventing-ci generate-serving-ci generate-serverless-operator-ci
-.PHONY: generate-ci-no-clean
-
-generate-eventing-ci:
```

Comment:
These cmd name changes will most likely break GitHub actions that execute generators periodically.

Reply:
I'll update the workflows in this repo. Are you aware of other cross-repo references to those targets?

Reply:
The actions are updated, and I restored those targets, as deprecated (they print a warn box when used).

Reply:
I'd defer name change to @mgencur or @pierDipi to chime in here. 
I'm not a huge fan to carry this change within "add config" PR, because it doesn't seem to me as necessary for adding event plugin config.